plasma_framework/contracts/src/exits/payment/controllers/PaymentStartInFlightExit.sol

```diff
@@ -233,6 +234,10 @@ library PaymentStartInFlightExit {
         require(exit.exitStartTimestamp == 0, "There is an active in-flight exit from this transaction");
     }
 
+    function verifyInFlightTxType(StartExitData memory exitData) private pure {
+        require(exitData.inFlightTx.txType == exitData.controller.supportedTxType, "Not the supported tx type of the exit game");
```

Comment:
Why not just `Unsupported transaction type` or 'Exit game does not support this transaction type' ? Same for standard exit.

Reply:
done